.gitignore

```diff
@@ -1,3 +1,5 @@
 /node_modules
 /vendor
 .env
+/public/css
```

Comment:
Even though it means we have to deal with reconciling the built files, we still like to keep them in version control, so please drop these lines and re-add those files. Thanks!

Reply:
Sounds good, thanks!